org.flightgear.FlightGear.metainfo.xml

```diff
@@ -7,10 +7,10 @@
 	<project_license>GPL-2.0</project_license>
 	<id>org.flightgear.FlightGear</id>
 	<name>FlightGear</name>
-	<summary>A free and highly sophisticated flight simulator</summary>
-	<summary xml:lang="pt_BR">Um simulador de voo gratuito e altamente sofisticado</summary>
-	<summary xml:lang="es">Un simulador de vuelo gratuito y altamente sofisticado</summary>
-	<summary xml:lang="it">Un simulatore di volo gratuito e altamente sofisticato</summary>
+	<summary>Free open-source flight simulator</summary>
```

Comment:
```suggestion
	<summary>Free and open-source flight simulator</summary>
```

Reply:
I didn't add "and" initially due to the 35 characters limit. :smiley: